src/controllers/action-queue/cleaners.ts

```diff
@@ -112,12 +112,13 @@ const asLoanWithdrawActionOp: Cleaner<LoanWithdrawActionOp> = asObject({
 })
 const asSwapActionOp: Cleaner<SwapActionOp> = asObject({
   type: asValue('swap'),
-  fromWalletId: asString,
-  toWalletId: asString,
+  amountFor: asValue('from', 'to'),
+  walletId: asString,
```

Comment:
Why are we adding a third walletId?

Reply:
This is to support adding the AAVE network name to the swap description